content/docs/communities/moderation/_index.md

```diff
@@ -95,21 +93,280 @@ in the message bar at the bottom:
 /op @yourMjolnirBot:example.com 100
 ```
 
-TODO screenshot of oping mjolnir
+The bot really does need those privileges to be able to actually kick and ban
+people from the rooms it protects, or change other people's power level. Keep in
+mind that whoever has access to Mjolnir credentials effectively has powerlevel
+100 in those rooms. In this configuration Mjolnir is de-facto the owner of your
+community rooms. Some more advanced configurations allow you to keep more
+privileges than mjolnir, but they are quite tedious to set-up. We will not cover
+in this documentation.
 
 ## Getting moderation reports
 
-- You need your own instance while Aristotle isn't merged in the spec and
-  deployed in the wild?
+By default, the moderation reports go to the reporter's homeserver
+administrator. This is suboptimal for a variety of reasons. If the person
+running mjolnir and the person running the homeserver are the same, mjolnir can
+intercept some calls to display reports in its command room instead. Such
+reports look like below.
+
+![](./mjolnir_report.png)
+
+You can see the various details of the reported message, and handful of buttons
+below the message you can click on. These buttons will instruct mjolnir to take
+action, depending on which one you clicked.
+
+To enable moderation reports in Mjolnir's command room, please follow Mjolnir's
+[readable abuse reports documentation](https://github.com/matrix-org/mjolnir#enabling-readable-abuse-reports).
+
+When someone is obviously not behaving properly, you want to take action to make
+sure your community remains a nice place to hang out. To do so, you can do one
+of the following.
+
+## Enforcing moderation
+
+### Redacting a specific message
+
+If you have moderation reports enabled and click on the
+{{ mjolnirbutton(text="🗍 Redact") }} button below it, mjolnir will redact this
+message. Alternatively, if you don't have reports enabled or want to redact a
+specific message from mjolnir, you can instruct mjolnir to do so by sending a
+message in mjolnir's control room.
+
+To redact a specific message, you need to find its permalink. A permalink is a
+regular matrix.to link that you can obtain by hovering the message you want to
+delete, and clicking on the `…`, then on `Share`.
+
+![](./share_message.png)
+
+This will open a pop-up window from which you can copy the matrix.to permalink.
+
+![](./copy_permalink.png)
+
+Once you have the permalink, you can use it in mjolnir's redaction command
+
+```
+!mjolnir redact <event permalink> 
+```
+
+For example
+
+```
+!mjolnir redact https://matrix.to/#/!yOatELRSQXzfQMmxjH:matrix.org/$F76L2figPEC240TFaUkHoKPxxhJ3P54vP4hi14Sd8xw?via=matrix.org&via=t2bot.io
+```
+
+You may wonder why you should use mjolnir if you can simply redact the message
```

Comment:
Maybe it should be mentioned that redactions from mjolnir primarily are useful for mass redactions? As in redacting every message of a user. Since mjolnir is set up to not be rate limited it usually doesn't fail like a client would.

Reply:
Ah, I thought this was made obvious in the intro paragraph. Do you think I missed something? Would you tweak it to make it more visible?